src/tests/ftest/dfuse/sparse_file.py

```diff
@@ -94,14 +94,14 @@ def test_sparsefile(self):
         fsize_write_1024thwrite = get_remote_file_size(self.hostlist_clients[0], sparse_file)
         self.log.info("File size (in bytes) after writing 1024th byte: %s", fsize_write_1024thwrite)
         # verify file did not got overwritten after dd write.
-        self.assertTrue(fsize_write_1024thwrite == self.space_before)
+        self.assertEqual(fsize_write_1024thwrite, self.space_before)
```

Comment:
Could we add a descriptive message here that will be displayed if the assert fails?

Reply:
From a quick look, maybe
"File size incorrect after write"